codecov.yml

```diff
@@ -0,0 +1,22 @@
+# https://docs.codecov.com/docs/
+# https://docs.codecov.com/docs/codecov-yaml
+
+codecov:
+  branch: main # Set the default branch
+
+parsers:
+  go:
+    partials_as_hits: true
+
+ignore:
+  - "go/cmd/vttestserver"  # This relies on end-to-end test packages
```

Comment:
Can we also add the following paths to the `ignore` field? If so, would that allow us to remove the `rm -rf` command we have in the codecov GH Actions workflow
- `go/test/endtoend`
- `go/*/endtoend`
- `go/vt/*/endtoend`

Reply:
No, because the ignore field is a `codecov` directive but we temporarily remove those files so that `go test` doesn't try to execute all of the endtoend tests in addition to all of the unit tests.

Reply:
I am going to add those paths here as well, however, as codecov may then NOT count those lines as untested lines for the project (which they shouldn't be or we'll be in a never-ending cycle of testing the tests).